### ISSUE TITLE
Deck and tide error hourly communicating with each other

### ISSUE THREAD

**Issue by fejta** [kubernetes/test-infra#15492]
Any idea why this would happen so frequently?

deck:
```yaml
{
 insertId: "12atyw6fdt4pby"  
 jsonPayload: {
  agent: "tide"   
  component: "deck"   
  error: "[read tcp 10.108.9.37:57486->10.111.254.160:80: read: connection reset by peer]"   
  file: "prow/cmd/deck/tide.go:84"   
  func: "main.(*tideAgent).start.func2"   
  level: "error"   
  msg: "Updating history."   
 }
 labels: {…}  
 logName: "projects/k8s-prow/logs/deck"  
 receiveTimestamp: "2019-12-05T15:50:16.290523279Z"  
 resource: {…}  
 severity: "ERROR"  
 timestamp: "2019-12-05T15:50:12Z"  
}
```

tide

```yaml
{
 insertId: "1ny2s48g34llfqh"  
 jsonPayload: {
  component: "tide"   
  error: "write tcp 10.108.8.195:8888->10.108.9.37:57486: write: connection reset by peer"   
  file: "prow/tide/history/history.go:172"   
  func: "k8s.io/test-infra/prow/tide/history.(*History).ServeHTTP"   
  level: "error"   
  msg: "Writing JSON history response."   
 }
 labels: {…}  
 logName: "projects/k8s-prow/logs/tide"  
 receiveTimestamp: "2019-12-05T15:50:17.437534510Z"  
 resource: {…}  
 severity: "ERROR"  
 timestamp: "2019-12-05T15:50:12Z"  
}
```

/area prow
/area prow/tide
/area prow/deck
/assign @fejta @Katharine @cjwagner 
/kind oncall-hotlist

**Comment by stevekuznetsov** [comment]
Do you run the descheduler or autoscaler on your worker node pool? Could these Pods be bouncing around Nodes?

**Comment by fejta** [comment]
No

Possible. I checked the pod and didn't see any restarts, but didn't check the deployment.

**Comment by cjwagner** [comment]
Duplicate of this issue: https://github.com/kubernetes/test-infra/issues/14520
Alvaro suggested that this might be fixed by k8s version 1.15+ https://github.com/kubernetes/test-infra/issues/14520#issuecomment-536759681
